### ISSUE TITLE
Add Scylla Operator to https://operatorhub.io/

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla-operator#43]
https://operatorhub.io/ holds many open-source operators for user to use.
It will be useful to add Scylla to this list once it is ready

A next step can be to certify the operator 
https://connect.redhat.com/explore/red-hat-openshift-operator-certification

**Comment by dyasny** [comment]
I don't think it was ever tested with openshift, openshift is based on k8s, but it also has some differences

**Comment by dahankzter** [comment]
Operatorhub seems "standard" so that we should do. As for Openshift I think we can count on the internals being kubernetes compatible or it would break so many functional operators and charts which would make Openshift dead in the water. It needs testing of course like @dyasny suggested and some tweaks here and there I am sure.

**Comment by dyasny** [comment]
@dahankzter openshift defaults to specific networkig configs we might not have tested (weave/flanel instead of calico for example) and it typically uses CRI-O instead of docker, and that's just the tip of the iceberg there :)

**Comment by dahankzter** [comment]
This is out of scope for 1.0 at least I think @tzach since it seems to be very Openshift oriented.

**Comment by mykaul** [comment]
Depends on https://github.com/scylladb/scylla-operator/issues/424

**Comment by mflendrich** [comment]
Will be handled as milestone 3 - tracked in the epic-level item #2346.
